scss/components/_article.scss

```diff
@@ -0,0 +1,223 @@
+.article{
+    padding: 0 20px;
+    #article-container{
+        & > div:first-child{
```

Comment:
Don't rely on `:first-child` for styling. Each component automatically outputs a parents element with a child element with the class `cmp-{component_name}`. Target that child element instead. Please look for this in every SCSS file and make the necessary changes.